tools/msstatstmt/msstatstmt.xml

```diff
@@ -0,0 +1,364 @@
+            <filter>'ProfilePlot' in selected_outputs</filter>
+        </data>
+        <data name="out_qc_plot" from_work_dir="QCPlot.pdf" format="pdf" label="${tool.name} on ${on_string}: QC Plot">
+            <filter>'QCPlot' in selected_outputs</filter>
+        </data>
+        <data name="out_group_comp" from_work_dir="ComparisonResult.tsv" format="tsv" label="${tool.name} on ${on_string}: Group Comparison">
+            <filter>group['group_comparison'] == 'true'</filter>
+        </data>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="5">
+            <conditional name="input">
+                <param name="input_src" value="MSstats"/>
+                <param name="msstats_input" ftype="tabular" value="input.msstats.txt"/>
+            </conditional>
+            <param name="selected_outputs" value="msstats_log,msstatstmt_log,r_script,ProfilePlot,QCPlot"/>
+            <output name="out_msstats_log">
+                <assert_contents>
+                    <has_text text="1 level of Isotope type labeling in this experiment" />
+                </assert_contents>
+            </output>
+            <output name="out_msstatstmt_log">
+                <assert_contents>
+                    <has_text text="MSstatsTMT - proteinSummarization function" />
+                </assert_contents>
+            </output>
+            <output name="out_r_script">
+                <assert_contents>
+                    <has_n_lines n="53" />
+                </assert_contents>
+            </output>
+            <output name="out_profile_plot" file="ProfilePlot.pdf" compare="sim_size"/>
+            <output name="out_qc_plot" file="QCPlot.pdf" compare="sim_size"/>
+        </test>
+        <test expect_num_outputs="2">
+            <conditional name="input">
+                <param name="input_src" value="MaxQuant"/>
+                <param name="evidence" ftype="tabular" value="evidence.txt"/>
+                <param name="annotation" ftype="tabular" value="annotation.txt"/>
+                <param name="proteinGroups" ftype="tabular" value="proteinGroups.txt"/>
+            </conditional>
+            <conditional name="group">
+                <param name="group_comparison" value="true"/>
+                <conditional name="use_comp_matrix">
+                    <param name="select" value="true"/>
+                    <param name="comparison_matrix" ftype="tabular" value="comparison_matrix.txt"/>
+                </conditional>
+            </conditional>
+            <param name="selected_outputs" value="ProfilePlot"/>
+            <conditional name="which_Protein">
+                <param name="select" value="list"/>
+                <param name="protein_list" ftype="tabular" value="proteinIDs.txt"/>
+            </conditional>
+            <output name="out_group_comp" file="ComparisonResultMQ.tsv" compare="sim_size"/>
```

Comment:
sim_size for a tsv is not good. What is the problem with those test? Would it be better to use asserts? e.g. how many columns, rows, maybe check if some values are contained?